### PR TITLE
monalisa.sh

### PR DESCRIPTION
**Pull request by iraklic**
Adding a recipe for MonALISA for automatic deployment to CVMFS

### CODE REVIEW

File: monalisa.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package: MonALISA
+version: "20210818"
+# tag: "1.0.0"
+# source: https://gitlab.cern.ch/jalien/jalien.git
+requires:
+ - JDK
+ - "curl:(?!slc8)"
+ - "system-curl:slc8.*"
+valid_defaults:
+ - monalisa
```
Comment (ktf) [inline monalisa.sh:10]: Please, do not introduce another default.

File: monalisa.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0 JDK/$JDK_VERSION-$JDK_REVISION \\
+            XRootD/$XROOTD_VERSION-$XROOTD_REVISION \\
+            xjalienfs/$XJALIENFS_VERSION-$XJALIENFS_REVISION
+# Our environment
+set JALIEN_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+prepend-path CLASSPATH \$JALIEN_ROOT/lib/alien-users.jar
+prepend-path PATH \$JALIEN_ROOT/bin
+EoF
```
Comment (ktf) [inline monalisa.sh:37]: I think you should use `alibuild-generate-module --bin --lib > etc/modulefiles/$PKGNAME` and add a `build_requires` on `alibuild-recipe-tools`.

File: monalisa.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package: MonALISA
+version: "20210818"
+# tag: "1.0.0"
+# source: https://gitlab.cern.ch/jalien/jalien.git
+requires:
+ - JDK
+ - "curl:(?!slc8)"
```
Comment (ktf) [inline monalisa.sh:7]: Do you really need curl at runtime? Also, given you only need it for building, I would simply drop the dependency.

**Review by ktf — CHANGES_REQUESTED** [review]
Please adapt the recipe following the suggestions.

File: monalisa.sh (hunk 1 above)
Comment (iraklic) [reply monalisa.sh:10]: Sorry, this looks weird, I will have to look up how to properly use github suggestions, I have not committed a suggestion before.

Comment (iraklic) [reply monalisa.sh:10]: somehow your comment doesn't show as a suggestion and mine is little messed up :)

**Comment by ktf** [comment]
Please have a look at #3342. For some reason I could not update this one.

**Comment by iraklic** [comment]
I see you have put in the changes that you suggested. Looks foot do me. I approve that recipe.
Is that PR going to be merged along with the one in `ali-bot` and this one dropped?

**Pull request closed without merging**